Simplified/Book/Models/NYPLBookRegistry.m

```diff
@@ -712,6 +722,78 @@ - (void)replaceBookmark:(NYPLReadiumBookmark *)oldBookmark with:(NYPLReadiumBook
   }
 }
 
+#if FEATURE_AUDIOBOOKS
+- (NSArray<NYPLAudiobookBookmark *> * _Nonnull)audiobookBookmarksForIdentifier:(NSString * _Nonnull)identifier {
```

Comment:
The logic of the functions related to `NYPLAudiobookBookmark` is nearly identical to the one of `NYPLReadiumBookmark`. Ideally we should have a generic function to handle both bookmark types, but since `NYPLAudiobookBookmark` is a forward declaration in the header file, making it very difficult to implement a generic function in ObjC.